std/experimental/logger/core.d

```diff
@@ -1424,7 +1428,7 @@ private @property Logger defaultSharedLoggerImpl() @trusted
         return emplace!FileLogger(buffer, stderr, LogLevel.info);
     }());
 
-    return stdSharedDefaultLogger;
+    return atomicLoad(cast(shared)stdSharedDefaultLogger);
```

Comment:
Might make sense to make `stdSharedDefaultLogger` `shared` instead of `__gshared` now.

Reply:
I'm not sure, shouldn't it be shared with everything hence __gshared!?

Reply:
`__gshared` is a hack in the type system for C style globals, since this is moving to `shared`, it should probably be `shared`.

Reply:
hm, but if this is D's logger and we are working to easily consume C, should we make sure we can use the logger from C leading to __gshared?

Reply:
No, it symbol  is available through the value returned by a function. 
If for some reason you need to access this from C, you should do so with `sharedLog()`, as you would  in D, not with the static storage within the function `sharedLog`.

Reply:
initOnce does not accept shared, not for the life of me.

Reply:
The difference between `shared` and `__gshared` is simply that the former types the value as `shared`. That's it. Other than that, they are identical.  `__gshared` does not make it accessible to C, that would be `extern(C)`, which would stop the name mangling from happening.

Reply:
> initOnce does not accept shared, not for the life of me.

You need to return `shared(FileLogger)` from the lambda, probably with an explicit cast.